api/src/main/java/org/pmiops/workbench/api/ProfileController.java

```diff
@@ -67,6 +70,25 @@
   }
 
   @Override
+  public ResponseEntity<Void> createAccount(CreateAccountRequest request) {
+    directoryService.createUser(
+        request.getGivenName(), request.getFamilyName(), request.getUsername(),
+        request.getPassword()
+    );
+    return null;
```

Comment:
This is wrong. Does anyone know how I can force this to return a 201?

Reply:
So you definitely want to return a ResponseEntity; you can use ResponseEntity.ok() to return a 200.

Swagger-generated TypeScript doesn't like returning null, so @blrubenstein added an EmptyResponse thing for this reason:

https://github.com/all-of-us/workbench/blob/aba8b0db098752e1d834963da2199bc87ab4fc67/api/src/main/resources/workbench.yaml#L564

https://github.com/all-of-us/workbench/blob/da87e79111a0fbcfd941d0402fcbec8b2ca215c7/api/src/main/java/org/pmiops/workbench/api/ClusterController.java#L102

So you can change these methods to return ResponseEntity<EmptyResponse> instead. (Which I think is totally fine.)

I know you can also say in Swagger that 201 is a valid response, and you could use:

https://docs.spring.io/spring-framework/docs/current/javadoc-api/org/springframework/http/ResponseEntity.html#ResponseEntity-org.springframework.http.HttpStatus-

or 

https://docs.spring.io/spring-framework/docs/current/javadoc-api/org/springframework/http/ResponseEntity.html#ResponseEntity-T-org.springframework.http.HttpStatus-

to make a response entity with a 201 here. I haven't tried that to see what gets returned or how the client generated code handles it, though.